### ISSUE TITLE
[PR] Add cornerRadius to ApplePayButtonView

### ISSUE THREAD

**Issue by AymenFarrah** [google-pay/flutter-plugin#127]
Hello,

We can customize the Apple Pay Button with a cornerRadius (by default set to 4.0)

So to do this we need to add when create the button :

```
let buttonType = arguments["type"] as! String
let buttonStyle = arguments["style"] as! String
let cornerRadius = arguments["cornerRadius"] as! CGFloat
```

then create the button with this method :

```
createApplePayView(type: buttonType, style: buttonStyle, cornerRadius: cornerRadius)
```

and finally customize it only when available iOS 12 minimum :

```
// Create the PK objects
    let paymentButtonType = PKPaymentButtonType.fromString(buttonTypeString) ?? .plain
    let paymentButtonStyle = PKPaymentButtonStyle.fromString(buttonStyleString) ?? .black
    let applePayButton = PKPaymentButton(paymentButtonType: paymentButtonType, paymentButtonStyle: paymentButtonStyle)

    if #available(iOS 12.0, *) {
        applePayButton.cornerRadius = cornerRadius
    }
```

**Comment by JlUgia** [comment]
Hi @AymenFarrah, thank you for the suggestion.
Feel free to suggest a PR with the changes and we'll take a look.
If you are not able to do so, let us know too.

**Comment by JlUgia** [comment]
Hello @AymenFarrah, PRs are accepted from external forks.
Please take a look at the [contribution guide](https://github.com/google-pay/flutter-plugin/blob/main/CONTRIBUTING.md) before proposing any changes.

**Comment by JlUgia** [comment]
Published under [`1.2.0-beta02`](https://pub.dev/packages/pay/versions/1.2.0-beta02).
